### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by derp-cas**
added notes to notes

### CODE REVIEW

File: multi-purpose-tool/notes.txt [hunk 1]
```diff
@@ -11,4 +11,5 @@ Goal:
 	- TypeScript -> Later upgrade
 	- StoryBook library
 	- Jest
-	- React-testing-library
\ No newline at end of file
+	- React-testing-library
+	Was geht
```
Comment (Artur-SE) [inline multi-purpose-tool/notes.txt]: Ey, bist du behinder? WTF? Mach mal weg

File: multi-purpose-tool/notes.txt [hunk 2]
```diff
@@ -11,4 +11,5 @@ Goal:
 	- TypeScript -> Later upgrade
 	- StoryBook library
 	- Jest
-	- React-testing-library
```
Comment (Artur-SE) [inline multi-purpose-tool/notes.txt:14]: What is this shit?

File: multi-purpose-tool/notes.txt [hunk 3]
```diff
@@ -11,4 +11,5 @@ Goal:
 	- TypeScript -> Later upgrade
```
Comment (Artur-SE) [inline multi-purpose-tool/notes.txt:11]: test

File: multi-purpose-tool/notes.txt (hunk 2 above)
Comment (derp-cas) [reply multi-purpose-tool/notes.txt:14]: i dont know man

**Review by Artur-SE — COMMENTED** [review]
@derp-cas ey, was geht

File: multi-purpose-tool/notes.txt (hunk 1 above)
Comment (derp-cas) [reply multi-purpose-tool/notes.txt]: Ok Chef!

**Comment by Artur-SE** [comment]
done

**Comment by Artur-SE** [comment]
done

**Review by Artur-SE — COMMENTED** [review]
done

**Review by Artur-SE — APPROVED** [review]

**Pull request merged**